src/main/java/com/xiaomi/infra/pegasus/tools/interceptor/InterceptorManger.java

```diff
@@ -0,0 +1,30 @@
+package com.xiaomi.infra.pegasus.tools.interceptor;
+
+import com.xiaomi.infra.pegasus.base.error_code.error_types;
+import com.xiaomi.infra.pegasus.rpc.async.ClientRequestRound;
+import com.xiaomi.infra.pegasus.rpc.async.TableHandler;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InterceptorManger {
+
+  private List<TableInterceptor> interceptors = new ArrayList<>();
+
+  public InterceptorManger add(TableInterceptor interceptor) {
+    interceptors.add(interceptor);
+    return this;
+  }
+
+  public void executeBefore(ClientRequestRound clientRequestRound, TableHandler tableHandler) {
```

Comment:
```suggestion
  public void interceptBefore(ClientRequestRound clientRequestRound, TableHandler tableHandler) {
```

Why is this method not consistent with `TableInteceptor`?